index.rst

```diff
@@ -152,19 +151,19 @@ valid values for these properties:
      - Set to ``latin1``, ``utf-8``, ``utf-8-bom``, ``utf-16be`` or ``utf-16le`` to
        control the character set. Use of ``utf-8-bom`` is discouraged.
    * - ``trim_trailing_whitespace``
-     - Set to ``true`` to remove any whitespace characters preceeding newline
-       characters and ``false`` to ensure it doesn't.
+     - Set to ``true`` to remove all whitespace characters preceeding newline
+       characters in the file and ``false`` to ensure it doesn't.
    * - ``insert_final_newline``
      - Set to ``true`` ensure file ends with a newline when saving and ``false``
        to ensure it doesn't.
    * - ``root``
-     - Must be specified at the top of the file outside of any sections. Set
-       to ``true`` to stop ``.editorconfig`` files search on current file. The
-       value is case insensitive.
+     - Must be specified in the default (global/root) section of the file. Set
```

Comment:
We have defined section as "the lines starting from a Section Header until the beginning of the next Section Header or the end of the file." That's why I think it should be "outside of any sections." The statement before the change is also consistent with "Any property other than ``root`` MUST be
located under a section to take effect."

Reply:
Good point. The root section is still considered a section though. I've seen it referred to as the default section.

What do you think if we go the other way around and instead redefine a section to be:

> A collection of properties, comments and blank lines; typically, including a section header; the only exception being the default (i.e., global/root) section, which _never_ includes a section header.

I'd be happy to create a graphic to illustrate "The Anatomy of an EditorConfig INI File" in order to suppliment this documentation and add even more clarity.

Reply:
Since we have mentioned so many times of section within the document, do you think it's just easier to call it something different, such as preamble?

> Preamble: From the beginning to the first Section Header.

Reply:
I was going to say no, but the more I look into the AST interfaces, that does actually make a lot of sense.

## INI parsing

The preamble and the collection of sections that follow:

- Are siblings!
- May contain blank lines, comments and any number of properties.

## EditorConfig-Specific Rules

- The preamble may only contain a single property of `root=true`.

Reply:
Uhm - i am sure i missed that but i use the preamble to define "global" defaults. Is that considered being bad behaviour and should be done below an explicit asterisk-section?

Reply:
Can you give me an example INI file for context?

Reply:
No i can't - after crawling through my ecfg-files i found out i consistently use the `[*]`-section. I shouldn't write comments while being distracted.. Sorry. :boy: :girl:

Reply:
All good, my friend.